src/core.js

```diff
@@ -441,7 +441,6 @@ $.extend( $.validator, {
 			if ( errors ) {
 				// add items to error list and map
 				$.extend( this.errorMap, errors );
-				this.errorList = [];
```

Comment:
still not sure this is the right thing todo. could we clear the error list and add all elemens from the `errorMap` into the list? I mean the errorMap contains everything but the errorList doesn't, right?